### ISSUE TITLE
Integration tests assumes some folder structure

### ISSUE THREAD

**Issue by ruigulala** [istio/istio#28371]
**Bug description**
Without setting REPO_ROOT, Integration tests assumes some folder structure when you run it.
The tests will fail with the following error if this implicit requirement is not satisfied.
```
Default IOPFile missing: stat /usr/local/google/home/ruigu/Workspace/istio-oss/tests/integration/pilot/tests/integration/iop-integration-test-defaults.yaml: no such file or directory
```

[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**

**Steps to reproduce the bug**
```
TAG=${TAG:-latest} HUB=${HUB:-gcr.io/istio-testing} go test -tags=integ -v -p 1 ./tests/integration/pilot/ -run TestTraffic
```
**Version (include the output of `istioctl version --remote` and `kubectl version --short` and `helm version` if you used Helm)**

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by Monkeyanator** [comment]
I've run into this as well. Had to change the `REPO_ROOT` environment variable to get things working.

**Comment by howardjohn** [comment]
fixed by #29050
